src/program.js

```diff
@@ -6,6 +6,8 @@ import {readFileSync} from 'fs';
 import camelCase from 'camelcase';
 import decamelize from 'decamelize';
 import yargs from 'yargs';
+// TODO(rpl): try to remove the following suppress comment after updating flow to more recent versions.
+// $FlowFixMe: flow doesn't seem to read yet the package.json config that should make yargs/yargs available.
```

Comment:
What do you mean by "the package.json config"? Do you mean the "exports" declaration in package.json? Be more explicit so that this comment is more actionable.